ACPs/181-proposervm-epochs/README.md

```diff
@@ -0,0 +1,148 @@
+| ACP | 181 |
+| :--- | :--- |
+| **Title** | P-Chain Epoched Views |
+| **Author(s)** | Cam Schultz [@cam-schultz](https://github.com/cam-schultz) |
+| **Status** | Proposed, Implementable, Activated, Stale ([Discussion](POPULATED BY MAINTAINER, DO NOT SET)) |
+| **Track** | Standards |
+
+## Abstract
+
+Proposes a standard P-Chain epoching scheme such that any VM that implements it has consistent view of the P-Chain for a known duration of time. This would enable VMs to optimize validator set retrievals, which currently must be done as often as every P-Chain block. This standard does *not* introduce epochs to the P-Chain's VM directly. Instead, it provides a standard that may be implemented by layers that inject P-Chain state into VMs, such as the ProposerVM.
```

Comment:
```suggestion
Proposes a standard P-Chain epoching scheme such that any VM that implements it has a consistent view of the P-Chain for a known duration of time. This would enable VMs to optimize validator set retrievals, which currently must be done as often as every P-Chain block. This standard does *not* introduce epochs to the P-Chain's VM directly. Instead, it provides a standard that may be implemented by layers that inject P-Chain state into VMs, such as the ProposerVM.
```

Reply:
Bumping this suggestion.